### DISCUSSION TITLE
Old config revision pops up every time after upgrade

### DISCUSSION THREAD

**Discussion by markkuleinio** [netbox-community/netbox#14178] [Help Wanted!] [answered]
I installed NetBox 3.6.1 (or so) in a new test VM earlier this fall and dumped the database from the earlier 3.3.x installation. I've since then upgraded this test instance a couple of times, now to 3.6.4. After each upgrade NetBox reverts back to the config revision number 1:

![image](https://github.com/netbox-community/netbox/assets/25136274/007fb5b6-3e8e-430e-96b9-ec5a18dddc35)

I then need to restore the latest revision, and settings are fine again. Until the next upgrade, again rev 1 is activated.

Is this something that someone else has experienced as well?

**Comment by julianstolp** [comment, marked as answer]
I can confirm this behavior and opened a ticket #14182

**Reply by markkuleinio** [reply]
Thanks! Just upgraded to 3.6.5 and the latest config revision stayed active as expected.
